pi_controller/README.md

```diff
@@ -0,0 +1,31 @@
+# Raspberry Pi
+
+## Functions
+
+The Raspberry Pi in Cat Town performs a few key functions:
+
+* Runs computer vision to detect the cat's location;
+* Simulates mouse personalities to decide where the mice will run next;
+* Decides when to deploy mice and which ones;
+* Serves a user interface over the Web for remote viewing and control.
```

Comment:
This might be encapsulated in "user interface," but two other functions we could spell out here are:
- Detect user reconfiguration of the board (by way of reading occasional QR codes on platforms)
- Detect or take user input when user replaces a mouse on the board/cartridge